app/models/card.rb

```diff
@@ -4,21 +4,36 @@ class Card < ActiveRecord::Base
 
   before_validation :set_default_review_date, :strip_whitespaces, on: :create
 
+  scope :need_to_review, -> { where("review_date <= ?", Time.zone.now ).order("RANDOM()")}
```

Comment:
Line is too long. [90/80]<br>Space missing inside }.<br>Space inside parentheses detected.